resources/react/README.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar: auto
+---
+
+# React
+
+## What is React
+React (also known as ReactJS or React.js) is an open-source declarative, efficient and flexiable JavaScript library for building user interface. 
+
+It lets you compose complex UIs from small and isolated pieces of code called "components". React can be used as a base in the development of single-page or mobile application.
```

Comment:
Explain what you mean by used as a base. Try finding some definitions written online as well.